domains/transfer/Shared/ProjectOriginClients.Tests/ProjectOriginClients.Tests.csproj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
+    <PackageReference Include="xunit" Version="2.8.1" />

```

Comment:
Are we sure we want to run these?

Reply:
I don't know... I found an unused test project, so I guess it's either run these or delete the project.